element_calcium_imaging/imaging_report.py

```diff
@@ -0,0 +1,414 @@
+import cv2
+import numpy as np
+import datajoint as dj
+from functools import partial
+from plotly.io import from_json
+import plotly.graph_objects as go
+from ipywidgets import widgets as wg
+from . import imaging, scan
+
+schema = imaging.schema
+
+
+def single_to_3channel_image(image, low_q=0, high_q=99.9):
```

Comment:
I'd suggest moving all of the plotting functions elsewhere, so users can call the plotting function independent of all of the DataJoint tables